campaign/record_martialadvantage.xml

```diff
@@ -117,4 +140,103 @@
 			</label_reftype>	
 		</sheetdata>
 	</windowclass>
+
+	<windowclass name="martialadvantages_actions">
+		<margins control="0,0,0,2" />
+		<script>
+			function onInit()    
+				update();
+			end
+
+			function onClose()
+			end
+
+			function updateControl(sControl, bReadOnly, bForceHide)
+				if not self[sControl] then
+					return false;
+				end
+				
+				return self[sControl].update(bReadOnly, bForceHide);
+			end
+
+			function update()
+				local bReadOnly = WindowManager.getReadOnlyState(getDatabaseNode());
+				test_iadd.setVisible(not bReadOnly)
+				damage_iadd.setVisible(not bReadOnly)
+				heal_iadd.setVisible(not bReadOnly)
+				effect_iadd.setVisible(not bReadOnly)
+
+				uniteffect_iadd.setVisible(not bReadOnly);
+				uniteffects.update();
+			end
+		</script>
+		<sheetdata>
+			<anchor_column name="columnanchor" />
+			<label_column name="actions_label">
+				<static textres="power_label_displayaction" />
+				<font>reference-b-large</font>
+				<anchored>
+					<top relation="relative" offset="10" />
+					<right/>
+					<left offset="5" />
+				</anchored>
+			</label_column>
+			<action_button name="effect_iadd">
+				<anchored>
+					<top parent="actions_label" anchor="top" />
+					<right anchor="right" offset="-10" />
+				</anchored>
+				<icon normal="button_action_effect" pressed="button_action_effect_down" />
+				<target>effect</target>
+			</action_button>
+			<action_button name="heal_iadd">
+				<anchored to="effect_iadd"/>
+				<icon normal="button_action_heal" pressed="button_action_heal_down" />
+				<target>heal</target>
+			</action_button>
+			<action_button name="damage_iadd">
+				<anchored to="heal_iadd" />
+				<icon normal="button_action_damage" pressed="button_action_damage_down" />
+				<target>damage</target>
+			</action_button>
+			<action_button name="test_iadd">
+				<anchored to="damage_iadd" />
+				<icon normal="button_action_attack" pressed="button_action_attack_down" />
+				<target>test</target>
+			</action_button>
+
+			<list_action name="actions">
+				<datasource>.actions</datasource>
+			</list_action>
+			<!-- <scrollbar>
```

Comment:
Probably at the point where we want to start cleaning out commented code.

Reply:
Yup, I cleaned out commented xml